<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportWindow</class>
 <widget class="QMainWindow" name="ReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Report Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:18px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:xx-large; font-weight:600;&quot;&gt;Project Manager Report Generator&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Welcome to the Project Manager Report Generator. You can generate reports regarding balance, project timelines etc using the menus.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Balance</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Balance for</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QComboBox" name="filterModeSelection">
                 <item>
                  <property name="text">
                   <string>Selected</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All Clients</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>All Projects</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="includeSubprojectsButton">
                 <property name="text">
                  <string>Include Subprojects</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QCheckBox" name="periodReportCheckbox">
               <property name="text">
                <string>For Projects Started between</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="fromEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>and</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="toEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Clients</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListView" name="clientList">
                 <property name="selectionMode">
                  <enum>QAbstractItemView::MultiSelection</enum>
                 </property>
                 <property name="selectionBehavior">
                  <enum>QAbstractItemView::SelectItems</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Client Projects</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QTreeView" name="projectList">
                 <property name="selectionMode">
                  <enum>QAbstractItemView::MultiSelection</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QCommandLinkButton" name="generateReportButton">
               <property name="text">
                <string>Generate report</string>
               </property>
               <property name="icon">
                <iconset resource="icons.qrc">
                 <normaloff>:/active</normaloff>:/active</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
               <property name="description">
                <string extracomment="Generate a report according to the selections above"/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Timeline</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuReports">
    <property name="title">
     <string>Reports</string>
    </property>
    <addaction name="actionProject_Report"/>
    <addaction name="actionClient_Report"/>
    <addaction name="actionBalance"/>
   </widget>
   <widget class="QMenu" name="menuQuick_Reports">
    <property name="title">
     <string>Quick Reports</string>
    </property>
    <addaction name="actionThis_Month_s_Projects"/>
    <addaction name="actionPending_Costs_Report"/>
    <addaction name="actionLast_Year_s_Projects"/>
   </widget>
   <addaction name="menuReports"/>
   <addaction name="menuQuick_Reports"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionProject_Report">
   <property name="text">
    <string>Project...</string>
   </property>
  </action>
  <action name="actionClient_Report">
   <property name="text">
    <string>Client...</string>
   </property>
  </action>
  <action name="actionThis_Month_s_Projects">
   <property name="text">
    <string>This Month's Projects</string>
   </property>
  </action>
  <action name="actionPending_Costs_Report">
   <property name="text">
    <string>Pending Costs Report by Client</string>
   </property>
  </action>
  <action name="actionLast_Year_s_Projects">
   <property name="text">
    <string>Last Year's Projects</string>
   </property>
  </action>
  <action name="actionBalance">
   <property name="text">
    <string>Pending Costs...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
